src/__tests__/PresentationVerifier.test.ts

```diff
@@ -1,5 +1,5 @@
-import _ from 'lodash';
 import { Credential } from '../Credential';
+import * as R from 'ramda';
```

Comment:
I think if you include @types/ramda you can import this "properly" i.e without the * as R,. But I might be wrong.

Reply:
you're right, updated